pkg/plugins/policies/meshtimeout/plugin/v1alpha1/plugin_test.go

```diff
@@ -0,0 +1,485 @@
+								MaxStreamDuration:     parseDuration("10m"),
+								MaxConnectionDuration: parseDuration("10m"),
+							},
+						},
+					},
+				},
+			},
+			expectedListener: "basic_outbound_listener.golden.yaml",
+			expectedCluster:  "basic_outbound_cluster.golden.yaml",
+		}),
+		Entry("tcp outbound route", sidecarTestCase{
+			resources: []core_xds.Resource{{
+				Name:   "outbound",
+				Origin: generator.OriginOutbound,
+				Resource: NewListenerBuilder(envoy_common.APIV3).
+					Configure(OutboundListener("outbound:127.0.0.1:10002", "127.0.0.1", 10002, core_xds.SocketAddressProtocolTCP)).
+					Configure(FilterChain(NewFilterChainBuilder(envoy_common.APIV3).
+						Configure(TcpProxy(
+							"127.0.0.1:10002",
+							envoy_common.NewCluster(
+								envoy_common.WithService("backend"),
+								envoy_common.WithWeight(100),
+							),
+						)),
+					)).
+					MustBuild(),
+			},
+				{
+					Name:     "outbound",
+					Origin:   generator.OriginOutbound,
+					Resource: clusterWithName("second-service"),
+				}},
+			toRules: core_xds.ToRules{
+				Rules: []*core_xds.Rule{
+					{
+						Subset: core_xds.Subset{core_xds.Tag{
+							Key:   mesh_proto.ServiceTag,
+							Value: "second-service",
+						}},
+						Conf: api.Conf{
+							ConnectionTimeout: parseDuration("10s"),
+							IdleTimeout:       parseDuration("30s"),
+						},
+					},
+				},
+			},
+			expectedCluster:  "basic_tcp_cluster.golden.yaml",
+			expectedListener: "basic_tcp_listener.golden.yaml",
+		}),
+		Entry("basic inbound route", sidecarTestCase{
+			resources: []core_xds.Resource{{
+				Name:     "inbound",
+				Origin:   generator.OriginInbound,
+				Resource: httpInboundListenerWith(80),
+			},
+				{
+					Name:     "inbound",
+					Origin:   generator.OriginInbound,
+					Resource: clusterWithName("localhost:80"),
+				}},
+			fromRules: core_xds.FromRules{
+				Rules: map[core_xds.InboundListener]core_xds.Rules{
+					{
+						Address: "127.0.0.1",
+						Port:    80,
+					}: []*core_xds.Rule{
+						{
+							Subset: core_xds.Subset{},
+							Conf: api.Conf{
+								ConnectionTimeout: parseDuration("10s"),
+								IdleTimeout:       parseDuration("1h"),
+								Http: &api.Http{
+									RequestTimeout:        parseDuration("5s"),
+									StreamIdleTimeout:     parseDuration("1s"),
+									MaxStreamDuration:     parseDuration("10m"),
+									MaxConnectionDuration: parseDuration("10m"),
+								},
+							},
+						},
+					}},
+			},
+			expectedCluster:  "basic_inbound_cluster.golden.yaml",
+			expectedListener: "basic_inbound_listener.golden.yaml",
+		}),
+		Entry("outbound with defaults when http conf missing", sidecarTestCase{
+			resources: []core_xds.Resource{{
+				Name:     "outbound",
+				Origin:   generator.OriginOutbound,
+				Resource: httpOutboundListenerWith(10001),
+			},
+				{
+					Name:     "outbound",
+					Origin:   generator.OriginOutbound,
+					Resource: clusterWithName("other-service"),
+				}},
```

Comment:
weird that formatter did not catch it. I'd place the last `{` from 209 in the new line
![image](https://user-images.githubusercontent.com/5459824/206702034-8891b5e3-6a4b-4ab0-a336-f52c1fe314cb.png)

both elements have the same meaning (element in the list), but indentation is different.

Reply:
fixed